ui/src/app/utils/access-utils.spec.tsx

```diff
@@ -928,3 +929,54 @@ describe(isCompleted.name, () => {
     expect(isCompleted(status, duccSignedVersion)).toBeFalsy();
   });
 });
+
+describe(syncModulesExternal.name, () => {
```

Comment:
I like how straightforward the tests are!

Reply:
Thanks!